nncf/experimental/onnx/engine.py

```diff
@@ -12,105 +12,40 @@
 """
 
 from typing import Dict
-import onnxruntime as rt
-import numpy as np
-import onnx
-from tqdm import tqdm
 
-from nncf.common.graph.transformations.commands import TargetType
-from nncf.common.graph.definitions import NNCFGraphNodeType
-from nncf.common.tensor import NNCFTensor
-from nncf.experimental.onnx.graph.onnx_graph import ONNXGraph
+import numpy as np
+import onnxruntime as rt
 
 from nncf.common.engine import Engine
-from nncf.common.graph.factory import NNCFGraphFactory
-from nncf.common.tensor_statistics.statistic_point import StatisticPointsContainer
-from nncf.experimental.onnx.tensor import ONNXNNCFTensor
 
 
 class ONNXEngine(Engine):
     """
     Engine for ONNX backend using ONNXRuntime to infer the model.
     """
 
-    def __init__(self, **rt_session_options):
-        super().__init__()
-        self._inputs_transforms = lambda input_data: input_data.astype(np.float32)
-        self.sess = None
+    def __init__(self, model, **rt_session_options):
```

Comment:
Is `model` even used after the refactoring anymore? If the model is required for the methods such as `infer`, why not make the model an argument of that method?

Reply:
Please see the lines below. The incoming model uses to create a session.
`serialized_model = model.SerializeToString()`